lib/trento/application/integration/checks/checks.ex

```diff
@@ -7,6 +12,52 @@ defmodule Trento.Integration.Checks do
   def request_execution(execution_id, cluster_id, hosts, selected_checks),
     do: adapter().request_execution(execution_id, cluster_id, hosts, selected_checks)
 
+  @spec get_catalog ::
+          {:ok, FlatCatalog.t()} | {:error, any}
+  def get_catalog do
+    case adapter().get_catalog() do
+      {:ok, catalog} ->
+        FlatCatalog.new(%{checks: catalog})
+
+      {:error, :not_ready} ->
+        {:error, "The catalog is still being built. Try again in some moments"}
+
+      {:error, reason} ->
+        {:error, reason}
+
+      _ ->
```

Comment:
there is no need for this match, blackholes hide cases and if we do not receive an :ok or an :error it is fine to break because there is something wrong in the code

Reply:
also the get_catalog doesnt return anything except :ok tuples and :error tuples so this case will never match